src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -868,6 +907,12 @@ void Transformations::MainSnippets(void) {
                 return false;
             },
             snippets::pass::TokenizeSnippets);
+            CPU_SET_CALLBACK_ARM(snippetsManager, [&](const std::shared_ptr<const ov::Node>& n) -> bool {
+                const bool is_support_op = ov::is_type<const ov::op::v1::Add>(n) ||
```

Comment:
We already have callback for x64 that contains common conditions for all archs (constants paths, ranks, dynamic op). Can we make this callback common and just add `is_supported_op` with different implementations for arhcs?
```
auto is_supported_op = [](node) {
#if defined arm
........
#else if defined x64
.....
#endif
};

CPU_SET_CALLBACK_COMMON(snippetsManager, [&](const std::shared_ptr<const ov::Node>& n) -> bool {
if (n->is_dynamic()) return true;
if (!is_supported_op(n)) return true;

// other checks
};
```

What do you think?

Reply:
Applied. Thanks Alexandra!